cl/people_db/import_judges/populate_fjc_judges.py

```diff
@@ -147,8 +147,14 @@ def add_positions_from_row(item, person, testing, fix_nums=None):
 
         if pd.isnull(item['Court Name' + pos_str]):
             continue
-        courtid = match_court_string(item['Court Name' + pos_str],
-                                     federal_district=True)
+
+        if re.search('appeal', item['Court Name' + pos_str], re.I):
```

Comment:
I have so many questions! How did this work for appellate before?!

Reply:
Again, don't know but maybe a CSV of only appellate judges or district court judges was read in and the values were changed accordingly?  That's the only thing I can think of...